stellar-dotnet-sdk/FeeBumpTransaction.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using stellar_dotnet_sdk.xdr;
+using Int64 = stellar_dotnet_sdk.xdr.Int64;
+
+namespace stellar_dotnet_sdk
+{
+    public class FeeBumpTransaction : TransactionBase
+    {
+        public FeeBumpTransaction(IAccountId feeSource, Transaction innerTx, long fee)
```

Comment:
should this constructor be internal to encourage use of `BuildFeeBumpTransaction`?

Reply:
I think you're right.